settings.py

```diff
@@ -0,0 +1,10 @@
+import os
+from dotenv import load_dotenv
```

Comment:
We don't need dotenv to give us the environment variables. For running locally we use docker so it supports `docker run --env-file=.env` and in the cloud they support injecting env vars anyway.